Misc/NEWS.d/next/Documentation/2023-05-28-21-00-05.gh-issue-89455.j4cg7J.rst

```diff
@@ -0,0 +1 @@
+Add missing documentation for the ``exceptions`` attribute of the :class:`traceback.TracebackException` class
```

Comment:
We typically add one news file for all changes in the PR. Here there are two.

Reply:
Am I supposed to join it into a single sentence or use a line per each changelog entry?

Reply:
Each PR is squashed to a single commit. There needs to be one news file with a summary of the change. This won't be in the release notes, it can be very brief (and doesn't need all the details).